services/py-api/src/server/schemas/response_schemas/schemas.py

```diff
@@ -31,3 +31,7 @@ def serialize_participant(self, participant: Participant) -> Dict[str, Any]:
     @field_serializer("team")
     def serialize_team(self, team: Team) -> Dict[str, Any]:
         return team.dump_as_json()
+
+
+class NumberOfRandomParticipantsResponse(BaseModel):
+    number: int
```

Comment:
Don't commit these things for the same reason as you should not commit and push the above changes.

Reply:
Fixed in 1bec98abe406eea6382a5453ac98f3e900d3ddd4